src/test_speechtotext_1.py

```diff
@@ -0,0 +1,21 @@
+import speechtotext
+import os 
```

Comment:
either convert this file to proper test_cases, or remove this file altogether. If the purpose is to show how to use the API, move this file to a samples folder and create a sample application in there, along with the relevant file.